allennlp/version.py

```diff
@@ -0,0 +1,6 @@
+_MAJOR = "0"
+_MINOR = "3"
+_REVISION = "1-unreleased"
+
+VERSION_SHORT = "{0}.{1}".format(_MAJOR, _MINOR)
+VERSION = "{0}.{1}.{2}".format(_MAJOR, _MINOR, _REVISION)
```

Comment:
Can you add `from allennlp.version import VERSION as __version__` to `allennlp.__init__.py`?  I think that's the only thing missing here.  No strong preference between `VERSION` and `__version__` in the `__init__.py`, it just seems like `__version__` is more common.  We could also just give both...